lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/main/java/io/lighty/applications/rcgnmi/module/RcGnmiAppException.java

```diff
@@ -10,6 +10,14 @@
 
 public class RcGnmiAppException extends Exception {
 
+    public RcGnmiAppException() {
```

Comment:
Remove this constructor. Exception should always contain at least message or original cause.